src/Tooltip.js

```diff
@@ -96,20 +96,22 @@ const propTypes = {
   top: PropTypes.bool,
   center: PropTypes.bool,
   left: PropTypes.bool,
-  right: PropTypes.bool
+  right: PropTypes.bool,
+  zIndex: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
 }
 
 const defaultProps = {
   position: 'bottom',
   color: 'text',
   bg: 'white',
   align: 'right',
-  theme: theme
+  theme: theme,
+  zIndex: 'auto'
 }
 
 const Tooltip = ({ children, ...props }) => {
   return (
-    <div style={{ position: 'relative' }}>
+    <div style={{ position: 'relative', zIndex: props.zIndex }}>
```

Comment:
I know this element was already using inline styles, but I think it would be good to use a styled-component here. If you want to make that change, that'd be cool, otherwise we can follow up in a separate PR

Reply:
If this were a styled-component, we could easily avoid passing a value for zIndex when it's undefined, e.g. `${props => typeof props.zIndex === 'number' ? { zIndex: props.zIndex } : null}`

Reply:
@ronhenderson Any updates on this? We can go ahead and merge it and follow up with another PR to move the styles to styled-components.